python2_nbd_client.py

```diff
@@ -52,17 +58,59 @@ def close(self):
             self._disconnect()
             self._closed = True
 
-    def _non_fixed_new_style_handshake(self, export_name):
+    def _upgrade_socket_to_TLS(self):
+        thesocket = self._s
+        context = ssl.SSLContext(ssl.PROTOCOL_TLS)
```

Comment:
Can you add  a comment what protocol version, you're trying to force here?
i.e. TLS_v1_2